<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Editable Tree Model</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>445</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="MyTreeView" name="view" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode" >
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="animated" >
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>445</y>
     <width>573</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="exitAction" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="insertRowAction" >
   <property name="text" >
    <string>Insert Row</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="removeRowAction" >
   <property name="text" >
    <string>Remove Row</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="insertColumnAction" >
   <property name="text" >
    <string>Insert Column</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I, C</string>
   </property>
  </action>
  <action name="removeColumnAction" >
   <property name="text" >
    <string>Remove Column</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R, C</string>
   </property>
  </action>
  <action name="insertChildAction" >
   <property name="text" >
    <string>Insert Child</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTreeView</class>
   <extends>QTreeView</extends>
   <header>mytreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tlist.qrc" />
 </resources>
 <connections/>
</ui>
